src/scss/_dark.scss

```diff
@@ -60,10 +60,6 @@
     background-color: $dark-mode-darken;
   }
 
-  .avatar {
```

Comment:
this is not the right way to fix that - default avatars now do not have dark mode at all
![image](https://user-images.githubusercontent.com/7117869/116922806-269e3700-ac56-11eb-935a-fc67d4b42d1e.png)